src/nmodl/nocpout.cpp

```diff
@@ -403,7 +398,7 @@ extern Memb_func* memb_func;\n\
 
 	if (nmodl_text) {
 		Lappendstr(defs_list, "\n"
-"#define NMODL_TEXT 1\n"
+"#define NMODL_TEXT 0\n"
```

Comment:
This could be fixed differently; the issue is that the generated code looks like
```c++
static const char* nmodl_file_text;
something_that_uses(nmodl_file_test);
static const char* nmodl_file_text = "...";
```
and we just need to replace the 1st line with the 3rd one.

Reply:
Resolved in 487ca61f2ef5c542c0010b18a23c1ad816a6654f.